source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml

```diff
@@ -58,13 +58,25 @@
         <div class="title">Plan de acción </div><textarea class="content-editable" data-bind="value: evaluation.actionPlanComment, enable: isEvaluationEditable()"></textarea>
       </div>
     </div>
-    <h4>Comentarios de la Evaluación</h4>
+    <h4>Comentarios de la Empresa</h4>
     <div class="comments-container comments-company-container">
       <textarea class="content-editable big-comment" data-bind="value: generalComment, enable: isEvaluationEditable()"></textarea>
     </div>
-    <h4>Comentarios del Evaluado</h4>
-    <div class="comments-container comments-company-container">
-      <textarea class="content-editable big-comment" data-bind="value: evaluatedComment, enable: evaluation.readyForDevolution && !evaluation.finished && userView == 3"></textarea>
+    <div class="evaluation-comments">
+      <h4>Comentarios de la Evaluación</h4>
+      <div>
+        <!-- ko foreach: califications-->
+        <div data-bind="if: show() && (owner < 2 ||(owner == 2 && comments))">
```

Comment:
Well, if you think that is it possible to move this kind of complex logic to the view model, I think that it will be easier to read and less error prone. but if you think that it is not possible, please check it twice and keep it as it is.